### DISCUSSION TITLE
Would it be possible for Godot team to distribute Mac launcher to bypass requirement to own Apple device?

### DISCUSSION THREAD

**Discussion by SilencedPerson** [godotengine/godot-proposals#4859] [Platforms]
Just a program used to run .pck files so the user doesn't need to own Mac to export for the platform?
the user would just specify the .pck file with command line option and that would be it.
It would not be possible to get on Appstore but it would allow for distributing the app on third party sites like Itch.io or Steam.
Is this legally feasible?

**Comment by Calinou** [comment]
Godot already lets you export to macOS from Windows and Linux, but notarizing the `.app` requires running macOS at some point. Notarizing the app is not a strict requirement, but it does make running easier since you don't have to do the "right-click > Open twice" dance anymore.

> Just a program used to run .pck files so the user doesn't need to own Mac to export for the platform?

While this is a clever idea, distributing a signed and notarized app with the intent to run other people's untrusted code leaves *us* liable to getting our certificate revoked. We'd be responsible for other people potentially distributing malware. I'm pretty sure the Apple terms of service have something about this already :slightly_smiling_face:

**Reply by SilencedPerson** [reply]
Thank you, I didn't know it was possible.
